torch/ao/quantization/fx/prepare.py

```diff
@@ -1256,26 +1273,27 @@ def insert_observers_for_model(
                             node, model, modules, graph, matches,
                             node_name_to_target_dtype, pattern, qhandler, is_qat)
                         if maybe_output_obs_node is not None:
-                            # Update users of original node to use the output observer
-                            # instead. For example, change
-                            #
-                            #           next_node
-                            #          /
-                            #   cur_node -> obs
-                            #
-                            # to
-                            #
-                            #                 next_node
-                            #                 /
-                            #   cur_node -> obs
-                            #
-                            # We need to save orig users before updating uses because
-                            # the list of users will change as we update uses
-                            orig_users = list(node.users.keys())
-                            for user_node in orig_users:
-                                if user_node is maybe_output_obs_node:
-                                    continue
-                                user_node.replace_input_with(node, maybe_output_obs_node)
+                            if not _is_custom_module_lstm(node, modules):
```

Comment:
also would be good to put in comments saying here we assume that for quantizable lstm module we assume the output is observed internally. since we'll revisit this when we want to properly support this